aspnetcore/host-and-deploy/health-checks/samples/2.x/HealthChecksSample/ReadinessPublisher.cs

```diff
@@ -17,24 +16,18 @@ public ReadinessPublisher(ILogger<ReadinessPublisher> logger)
             _logger = logger;
         }
 
-        public List<(HealthReport report, CancellationToken cancellationToken)> 
-            Entries { get; } = 
-                new List<(HealthReport report, 
-                    CancellationToken cancellationToken)>();
-
-        public Exception Exception { get; set; }
-
         public Task PublishAsync(HealthReport report, 
             CancellationToken cancellationToken)
         {
-            Entries.Add((report, cancellationToken));
-
-            _logger.LogInformation("{TIMESTAMP} Readiness Probe Status: {RESULT}", 
-                DateTime.UtcNow, report.Status);
-
-            if (Exception != null)
+            if (report.Status == HealthStatus.Healthy)
+            {
+                _logger.LogInformation("{TIMESTAMP} Readiness Probe Status: {RESULT}", 
```

Comment:
I'm really not a fan of this kind of sample... **because** we already have logging for health check results. This isn't something a user should ever have to write.

Reply:
Then perhaps let's just have it ... as you say ... do nothing here. We can just show the skeleton of it. I'll try it on the next commit so that we can see how it looks.

Reply:
🤔 ..... idk about this ...... How about this ... we can add this line right into the sample code for the `ReadinessPublisher`. We already say this in the `LivenessProbeStartup` code where this publisher is registered ...

> The readiness check might do a set of more expensive or time-consuming checks to determine if all other resources are responding.